deploy/helm/sumologic/conf/logs/otelcol/config.yaml

```diff
@@ -210,16 +183,6 @@ processors:
             value: .+
 {{ end }}
 
-{{ if .Values.sumologic.logs.container.enabled }}
-  filter/include_fluent_tag_containers:
-    logs:
-      include:
-        match_type: regexp
-        record_attributes:
-          - key: fluent.tag
-            value: containers\..+
-{{ end }}
-
 {{ if .Values.sumologic.logs.systemd.enabled }}
   filter/include_fluent_tag_host:
```

Comment:
Should we create a separate issue to remove these kinds of holdouts?

Reply:
not sure what you mean by it, but we should get rid of `fluent.tag` concept